plugins/chat/assets/javascripts/discourse/models/chat-message.js

```diff
@@ -13,7 +83,8 @@ ChatMessage.reopenClass({
   },
 
   _initReactions(args) {
-    args.reactions = EmberObject.create(args.reactions || {});
+    args._reactions = args.reactions || {};
```

Comment:
If I didn't do this there was a conflict with `@tracked reactions` and things did not work